bigquery/sync_query.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * [START complete]
+ * Command-line application to perform an synchronous query in BigQuery.
+ *
+ * This sample is used on this page:
+ *
+ *   https://cloud.google.com/bigquery/querying-data#syncqueries
+ */
+
+'use strict';
+
+// [START auth]
+// By default, gcloud will authenticate using the service account file specified
+// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
+// project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
+var gcloud = require('gcloud');
+
+// Get a reference to the bigquery component
```

Comment:
`// Get a reference to the bigquery component`

=>

`// Instantiate the bigquery client`

I've been changing other samples to use this wording